app/models/generic_event.rb

```diff
@@ -161,17 +161,10 @@ def self.relationship_attributes(attributes)
       @serializer ||=
         Class.new(GenericEventSerializer).tap do |serializer|
```

Comment:
How do you feel about renaming this to something more explicit e.g. `new_serializer_class` ?

Reply:
Fine by me :)